pkg/plugin/proto/RestoreItemAction.proto

```diff
@@ -7,6 +7,7 @@ message RestoreExecuteRequest {
     string plugin = 1;
     bytes item = 2;
     bytes restore = 3;
+    bytes originalitem = 4;
```

Comment:
Please capitalize the `I` in `originalItem`

Reply:
@nrb @skriss in other bits of the restore code, we've called something like this `fromBackup` - how do you feel about that vs `originalItem`?

Reply:
I think `fromBackup` is a little clearer here

Reply:
changed to `fromBackup`